packages/e2e-tests/specs/keyboard-navigable-blocks.test.js

```diff
@@ -97,34 +97,6 @@ const tabThroughBlockToolbar = async () => {
 	);
 	await expect( isFocusedRightAlignmentControl ).toBe( true );
 
-	// Tab to focus on the 'Bold' formatting button
```

Comment:
Reason why these are no longer expected: the formatting toolbar is only available one the rich text field has selection. The buttons are useless otherwise. In master, they are useless as well, but they display because there is lingering local selection state, so the toolbar thinks that the rich text field is selected. If you select the block for the first time in master, there will also be no formatting buttons. See #13598.